cohttp-lwt-unix/src/server.ml

```diff
@@ -67,7 +67,29 @@ let log_on_exn =
        (Unix.error_message error) func arg)
   | exn -> Logs.err (fun m -> m "Unhandled exception: %a" Fmt.exn exn)
 
-let create ?timeout ?backlog ?stop ?(on_exn=log_on_exn) ?(ctx=Net.default_ctx)
-    ?(mode=`TCP (`Port 8080)) spec =
-  Conduit_lwt_unix.serve ?backlog ?timeout ?stop ~on_exn ~ctx:ctx.Net.ctx
-    ~mode (callback spec)
+let safe error_handler callback spec flow () =
+  Lwt.catch
+    (fun () -> let ic, oc = Conduit_lwt.io_of_flow flow in callback spec flow ic oc)
+    (fun exn -> error_handler exn)
+
+let create
+  : type cfg t flow.
+     ?timeout:int
+  -> ?backlog:int
+  -> ?stop:unit Lwt.t
+  -> ?on_exn:(exn -> unit)
+  -> cfg
+  -> (_, flow) Conduit_lwt.protocol
+  -> (cfg, t, flow) Conduit_lwt.Service.service
+  -> _ -> unit Lwt.t
+  = fun ?timeout:_ ?backlog:_ ?(stop= fst (Lwt.wait ())) ?(on_exn=log_on_exn)
```

Comment:
As before, we are ditching `timeout` and` backlog`. Is this something to worry about? Should we just remove them?